### ISSUE TITLE
Training set for BIR task

### ISSUE THREAD

**Issue by EternalEvan** [XPixelGroup/DiffBIR#64]
Hi, you choose the ImageNet for training. Did you select the images larger than 512-size before training or take all images into training set and resize them to 512-size? Thanks for your early reply!

**Comment by 0x3f3f3f3fun** [comment]
We did not filter out those images larger than 512, but instead used the center crop method to produce 512-sized images 😄.

**Comment by YilanWang** [comment]
I found that in the paper, the resolution of imagenet data is set to 512x512, but as far as I know, the resolution of imagenet is 256x256. How do we handle it? Is it similar to upsampling? Thank you very much

**Comment by EternalEvan** [comment]
> I found that in the paper, the resolution of imagenet data is set to 512x512, but as far as I know, the resolution of imagenet is 256x256. How do we handle it? Is it similar to upsampling? Thank you very much

Yeah. I assume the author upsamples the images along the shorter edge and uses the center crop method.
